test/metabase/driver/postgres_test.clj

```diff
@@ -257,7 +257,20 @@
                                                                                :dimension    ["field-id" (mt/id :users :user_id)]}}}
                                            :parameters [{:type   "text"
                                                          :target ["dimension" ["template-tag" "user"]]
-                                                         :value  "4f01dcfd-13f7-430c-8e6f-e505c0851027"}]}))))))))
+                                                         :value  "4f01dcfd-13f7-430c-8e6f-e505c0851027"}]})))))
+      (testing "Check that we can filter by multiple UUIDs for SQL Field filters"
+        (is (= [[#uuid "4f01dcfd-13f7-430c-8e6f-e505c0851027" 1]
+                [#uuid "da1d6ecc-e775-4008-b366-c38e7a2e8433" 3]]
+               (mt/rows (qp/process-query {:database   (mt/id)
+                                           :type       :native
+                                           :native     {:query         "SELECT * FROM users WHERE {{user}}"
+                                                        :template-tags {:user {:name         "user"
+                                                                               :display_name "User ID"
+                                                                               :type         "dimension"
+                                                                               :dimension    ["field-id" (mt/id :users :user_id)]}}}
+                                           :parameters [{:type   "text"
+                                                         :target ["dimension" ["template-tag" "user"]]
+                                                         :value  ["4f01dcfd-13f7-430c-8e6f-e505c0851027" "da1d6ecc-e775-4008-b366-c38e7a2e8433"]}]}))))))))
```

Comment:
These lines are getting a [little long](https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#keep-lines-to-120-chars). How about this?

```suggestion
               (mt/rows
                (qp/process-query
                 (assoc (mt/native-query
                         {:query         "SELECT * FROM users WHERE {{user}}"
                          :template-tags {:user
                                          {:name         "user"
                                           :display_name "User ID"
                                           :type         "dimension"
                                           :dimension    ["field-id" (mt/id :users :user_id)]}}})
                        :parameters
                        [{:type   "text"
                          :target ["dimension" ["template-tag" "user"]]
                          :value  ["4f01dcfd-13f7-430c-8e6f-e505c0851027"
                                   "da1d6ecc-e775-4008-b366-c38e7a2e8433"]}])))))))))
```